eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -245,15 +262,28 @@ data ConwayGovEvent era
 
 instance EraPParams era => NFData (ConwayGovEvent era)
 
+data GovSignal era = GovSignal
+  { gsVotingProcedures :: !(VotingProcedures era)
+  , gsProposalProcedures :: !(OSet.OSet (ProposalProcedure era))
+  , gsCertificates :: !(SSeq.StrictSeq (TxCert era))
+  }
+  deriving (Generic)
+
+deriving instance (EraPParams era, Eq (TxCert era)) => Eq (GovSignal era)
+deriving instance (EraPParams era, Show (TxCert era)) => Show (GovSignal era)
+
+instance (EraPParams era, NFData (TxCert era)) => NFData (GovSignal era)
+
```

Comment:
`GovProcedures` also had `NoThunks`, so I suppose you could add this: 
```suggestion
instance (EraPParams era, NoThunks (TxCert era)) => NoThunks (GovSignal era)
```

Reply:
NoThunk is only needed for stuff that we keep in the ledger state.

So, `NoThunks` for `GovProcedures` was totally useless.